metadata-ingestion/src/datahub/ingestion/transformer/add_dataset_browse_path.py

```diff
@@ -13,6 +13,7 @@
 
 class AddDatasetBrowsePathConfig(ConfigModel):
     path_templates: List[str]
+    full_overwrite: bool = False
```

Comment:
how about renaming this to `replace_existing`

another option could be: `operation: Operation = Operation.APPEND` where Operation is an enum with values `APPEND`, `REPLACE` etc.

Reply:
I was not happy with `full_overwrite` neither... so I was waiting for some suggestion 😅 
`replace_existing` sounds good to me, I will update the PR